src/Castle.Windsor/MicroKernel/SubSystems/Configuration/DefaultConfigurationStore.cs

```diff
@@ -181,7 +181,8 @@ public IConfiguration[] GetInstallers()
 
 		public IResource GetResource(String resourceUri, IResource resource)
 		{
-			if (resourceUri.IndexOf(Uri.SchemeDelimiter) == -1)
+			var uriSchemeDelimiter = "://";
```

Comment:
Instead of hardcoding these values, is this perhaps something we should think about putting in the compatibility layer? You never know, it might end up being a thing again in subsequent releases of netstandard.